File: backend/src/models/Bairro.js
const { DataTypes } = require('sequelize');
const { sequelize } = require('../config/database');

const Bairro = sequelize.define('Bairro', {
    id: {
        type: DataTypes.INTEGER,
        primaryKey: true,
        autoIncrement: true
    },
    nome: {
        type: DataTypes.STRING,
        allowNull: false
    },
    regiao: {
        type: DataTypes.STRING
    },
    ativo: {
        type: DataTypes.BOOLEAN,
        defaultValue: true
    }
}, {
    tableName: 'bairros'
});

module.exports = Bairro;

File: python-zeladoria/app/models/bairro.py
from sqlalchemy import Column, Integer, String, Boolean
from sqlalchemy.orm import relationship
from app.database.database import Base

class Bairro(Base):
    __tablename__ = "bairros"

    id = Column(Integer, primary_key=True, index=True)
    nome = Column(String, nullable=False)
    regiao = Column(String)
    ativo = Column(Boolean, default=True)

    # Relacionamentos
    chamados = relationship("Chamado", back_populates="bairro")
